test/typescript/FooRefTest.tsx

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable */
```

Comment:
Do we still need these files in this test directory?

Reply:
I was using these to test some typings -- i think it's useful to have them. But I can kill them if you want.

Reply:
I'm fine either way.